.teamcity/settings.kts

```diff
@@ -78,10 +78,10 @@ project {
 		param("teamcity.git.fetchAllHeads", "true")
 
 		// e2e config decryption key references. See PCYsg-vnR-p2 for more info.
-		password("E2E_SECRETS_ENCRYPTION_KEY_AUG_28_24" , "credentialsJSON:d1948ba8-3da0-4e9a-8478-1823619afca7", display = ParameterDisplay.HIDDEN)
 		password("E2E_SECRETS_ENCRYPTION_KEY_OCT_15_24" , "credentialsJSON:03a11b61-0f32-4b1b-8c82-3c02a902b022", display = ParameterDisplay.HIDDEN)
```

Comment:
Should be fine, although the previous one is usually kept for a longer time (just in case, I guess, because I can't think of a better reason for that)

Reply:
Yeah, It think that's the reason.